CHANGELOG_PENDING.md

```diff
@@ -36,6 +36,7 @@ Friendly reminder, we have a [bug bounty program](https://hackerone.com/tendermi
 ### BREAKING CHANGES
 
 - CLI/RPC/Config
+  - [config] \#7930 Add new event subscription options and defaults. (@creachadair)
```

Comment:
I have edited this PR's description to refer to #7930.
So we think it's better that this changelog line refers to #9491.